config.h.in

```diff
@@ -113,7 +113,7 @@
 #undef WASMAPP
 
 /* Makes config variables conditionally static, only in non-debug builds, to allow for overriding them in unit-tests. */
-#undef CONFIG_STATIC
+#undef CONFIG_STATIC_TYPE
```

Comment:
I suppose we could undef and define under `#if defined(__COVERITY__)`, leaving this as-is.